sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1026,12 +1050,23 @@ def _extract_endpoints(stage,  # type: fn_api_runner_transforms.Stage
   class StateServicer(beam_fn_api_pb2_grpc.BeamFnStateServicer,
                       sdk_worker.StateHandler):
 
+    class CopyOnWriteListProtocol(Protocol):
```

Comment:
How about we rename this protocol to `Buffer` and the other to `PartitionableBuffer`?  `PartitionableBuffer` can inherit from `Buffer` and add the `partition()` method.  

We don't want to rename `append()` to `overlay()` because we want to use a `list` as one of our data structures that implements this protocol.

Reply:
SG